### ISSUE TITLE
The wanted reason should be put in the criminal history

### ISSUE THREAD

**Issue by AJCM-git** [space-wizards/space-station-14#24956]

**Comment by deltanedas** [comment]
it is when they get arrested

**Comment by AJCM-git** [comment]
> it is when they get arrested

Yeah, and they should get a mention in the history when they warrant arrest, something like "UNDER SUSPICION FOR: *reason*" or something and then when you get arrested the current "ARRESTED:" suffices

**Comment by deltanedas** [comment]
ye
